apps/www/.storybook/main.ts

```diff
@@ -0,0 +1,24 @@
+import type { StorybookConfig } from "@storybook/nextjs"
+
+const config: StorybookConfig = {
+  stories: ["../stories/**/*.stories.@(js|jsx|ts|tsx|mdx)"],
+  addons: [
+    "@storybook/addon-links",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    {
+      name: "@storybook/addon-styling",
+      options: {
+        postCss: true,
+      },
+    },
```

Comment:
Hey there 👋 

Luckily for you, `@storybook/nextjs` handles postcss configuration of storybook for you 🎉 

That means you can drop the addon options

```suggestion
    "@storybook/addon-styling",
```